sync_data/sync/tests.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Not sure how Django likes to deal with tests, but a test or two (especially for `walk`) would be great!

Reply:
Yeah, that's on the todo list